docs/src/main/mdoc/consumers.md

```diff
@@ -200,8 +200,8 @@ object ConsumerStreamExample extends IOApp {
   def run(args: List[String]): IO[ExitCode] = {
     val stream =
       KafkaConsumer.stream(consumerSettings)
-        .evalTap(_.subscribeTo("topic"))
-        .flatMap(_.stream)
+        .subscribeTo("topic")
+        .stream
```

Comment:
I feel like maybe we should rename `stream` to `records` to avoid having common patterns with `stream` being used twice with different meanings.

Reply:
Sounds reasonable to me. We could try to add new methods in a 3.0 without deletions of old methods.